server/notification-providers/signl4.js

```diff
@@ -0,0 +1,60 @@
+const NotificationProvider = require("./notification-provider");
+const axios = require("axios");
+const { UP, DOWN } = require("../../src/util");
+
+class SIGNL4 extends NotificationProvider {
+
+    name = "SIGNL4";
+
+    /**
+     * @inheritdoc
+     */
+    async send(notification, msg, monitorJSON = null, heartbeatJSON = null) {
+        const okMsg = "Sent Successfully.";
+
+        try {
+            let data = {
+                heartbeat: heartbeatJSON,
+                monitor: monitorJSON,
+                msg,
+            };
+            // Source system
+            data["X-S4-SourceSystem"] = "UptimeKuma";
+            const config = {
+                headers: {
+                    "Content-Type": "application/json"
+                }
+            };
+
+            // Monitor URL
+            let monitorUrl;
+            if (monitorJSON) {
+                monitorUrl = this.extractAdress(monitorJSON);
+            }
```

Comment:
I meant this

```suggestion
            const monitorUrl = this.extractAdress(monitorJSON);
```

Reply:
Thanks and changed - a bit differently, since I do not (mis)use the monitorURL as ID anymore.